lib/models/list.js

```diff
@@ -27,7 +27,17 @@ async function addList(name, creationDate, products) {
   }
   return list;
 }
+async function deleteList(id) {
+  const listCollection = await getCollection("lists");
+  const objectId = new ObjectID.createFromHexString(id);
+  const list = await listCollection.deleteOne({ _id: objectId });
+  if (!list) {
+    return null;
+  }
+  return list;
+}
 
 exports.readLists = readLists;
 exports.readList = readList;
 exports.addList = addList;
+exports.deleteList = deleteList;
```

Comment:
You can export this as a module 
```suggestion
module.exports = {readLists, readList, addList, deleteList};
```